.vib/memcached/goss/goss.yaml

```diff
@@ -0,0 +1,20 @@
+file:
+  /tmp:
+    mode: "2777"
+    filetype: directory
+    exists: true
+command:
+  {{- $key := printf "key_%s" (randAlpha 5) }}
+  {{- $value := printf "value_%s" (randAlpha 5) }}
+  {{- $port := .Vars.service.ports.memcached }}
+  manage-key:
+    exec: printf "set {{ $key }} 0 300 11\r\n{{ $value }}\r\nquit\n" | nc memcached {{ $port }} && printf "get {{ $key }}\r\nquit\n" | nc memcached {{ $port }}
```

Comment:
You can avoid installing `netcat` and running the container as root if you use the following sample code instead, which uses bash redirections to a TCP port:

```bash
$ printf "set test_key_2 0 300 11\r\nvalue_12345\r\nquit\n" | (exec 3<>/dev/tcp/memcached/11211; cat >&3; cat <&3; exec 3<&-)
STORED
```

From: https://stackoverflow.com/questions/32838811/alternative-for-netcat-utility

I think I'd rather use it (with a comment to the source) than the current approach, which comes with the said disadvantages.

Reply:
That's a really nice alternative and it indeed solves the same concerns I had. Applied thanks!